aws/provider.go

```diff
@@ -454,6 +454,7 @@ func Provider() terraform.ResourceProvider {
 			"aws_opsworks_user_profile":                    resourceAwsOpsworksUserProfile(),
 			"aws_opsworks_permission":                      resourceAwsOpsworksPermission(),
 			"aws_opsworks_rds_db_instance":                 resourceAwsOpsworksRdsDbInstance(),
+			"aws_organization":                             resourceAwsOrganization(),
```

Comment:
Its going to look ugly at first and Naming is Hard, but we should probably also rename this `aws_organizations_organization` for a few reasons:
* The AWS Organizations service supports multiple resources we'll want to manage, e.g. `aws_organizations_account`, `aws_organizations_policy`, this will keep everything bundled together nicely naming-wise
* To prevent confusion with other AWS services that might use the terminology "organization"
* If AWS ever creates a separate account organization management service/method

Ideally the resource function names will be updated to long form as well: `resourceAwsOrganizationsOrganization`, `resourceAwsOrganizationsOrganizationRead`, etc

Reply:
I think I handled all the other feedback you provided.  I'm not against doing this but it seemed like more work than I was willing to put in tonight.